lib/json.js

```diff
@@ -11,13 +11,22 @@ var shrink = require("./shrink.js");
 var string = require("./string.js");
 var utils = require("./utils.js");
 
+var nulll = primitive.constant(null);
```

Comment:
I wasn't sure if this is better than e.g.
```js
var nulll = arbitraryBless({
  generator: generator.constant(null),
  shrink: shrink.noop,
  show: show.def,
});
```

Also, suggestions for a better name than `null` welcome 😅